src/pages/predictions.vue

```diff
@@ -0,0 +1,416 @@
+          </b>
+        </div>
+        <div class="small text-center text-muted">
+          Data Source:
+          <cite>
+            <a href="https://github.com/CSSEGISandData/COVID-19">CSSEGISandData</a> /
+            <a href="https://github.com/NovelCOVID/API">NovelCOVID</a> /
+            <a href="https://github.com/pomber/covid19">Pomber</a>
+          </cite>
+        </div>
+      </div>
+    </footer>
+  </Layout>
+</template>
+
+
+<script>
+import moment from "moment";
+import Chart from "../components/Chart";
+import PredictionData from "../../static/all-country-predictions.json";
+
+let starredCountries = [];
+export default {
+  components: {
+    Chart
+  },
+  created() {
+    this.sortCountriesData();
+  },
+  mounted() {
+    window.$ = require("jquery");
+    let mainLoaderContainer = document.getElementById("chart-loader");
+    mainLoaderContainer.style.display = "none";
+    this.userFunctions();
+    setTimeout(() => {
+      $("#tab-choose .btn-country").click();
+    }, 500);
+  },
+  data() {
+    return {
+      sortedPredictedArray: this.sortedPredictedArray,
+      PredictionData
+    };
+  },
+  methods: {
+    sortCountriesData: function() {
+      let predictedData = PredictionData;
+      let sortedPredictedArray = [];
+      let totalPredictedCountArray = [];
+      let predictedCountriesArray = Object.keys(predictedData);
+      predictedCountriesArray.forEach(country => {
+        let predictedTimelineArray = Object.values(predictedData[country]);
+        let predictedDateArray = Object.keys(predictedData[country]);
+        predictedDateArray.forEach(e => {
+          moment(e).format("DD-MM-YY");
+        });
+        let predictedActiveCount =
+          predictedTimelineArray[predictedTimelineArray.length - 1].Infected +
+          predictedTimelineArray[predictedTimelineArray.length - 1].Recovered;
+        let latestDate = predictedDateArray[predictedDateArray.length - 1];
+        totalPredictedCountArray.push({
+          country: country,
+          recovered:
+            predictedTimelineArray[predictedTimelineArray.length - 1].Recovered,
+          deaths:
+            predictedTimelineArray[predictedTimelineArray.length - 1].Fatal,
+          confirmed:
+            predictedTimelineArray[predictedTimelineArray.length - 1].Infected +
+            predictedTimelineArray[predictedTimelineArray.length - 1]
+              .Recovered +
+            predictedTimelineArray[predictedTimelineArray.length - 1].Fatal,
+          count: predictedActiveCount,
+          lastUpdated: latestDate
+        });
+      });
+      sortedPredictedArray = totalPredictedCountArray.sort((a, b) =>
+        a.count < b.count ? 1 : b.count < a.count ? -1 : 0
+      );
+      this.sortedPredictedArray = sortedPredictedArray;
+      console.log(sortedPredictedArray);
+      this.generateGraphContent(sortedPredictedArray, predictedData);
+    },
+    generateGraphContent: function(sortedPredictedArray, predictedData) {
```

Comment:
Same as above extract this function into a separate file and re-use in both pages